frontend/src/libs/components/input/input.tsx

```diff
@@ -7,22 +7,21 @@ import {
 
 import { useFormController } from '~/libs/hooks/hooks.js';
 
+import styles from './styles.module.scss';
+
 type Properties<T extends FieldValues> = {
   control: Control<T, null>;
   errors: FieldErrors<T>;
   label: string;
   name: FieldPath<T>;
-  placeholder?: string;
-  type?: 'text' | 'email';
-};
+} & React.ComponentPropsWithoutRef<'input'>;
```

Comment:
```suggestion
};
```
provide props that you need